test/fixtures/typescript/objects.spec.ts

```diff
@@ -1,6 +1,96 @@
 import { testFixture } from '../Fixture'
 
 describe('typescript objects tests', () => {
+  describe('simple object with trailing semicolon separator', () => {
+    testFixture({
+      input: '{ object: string; }',
+      stringified: '{object: string}',
+      diffExpected: {
+        typescript: {
+          type: 'JsdocTypeObject',
+          meta: {
+            separator: 'semicolon'
+          },
+          elements: [
+            {
+              type: 'JsdocTypeKeyValue',
+              key: 'object',
+              right: {
+                type: 'JsdocTypeName',
+                value: 'string'
+              },
+              optional: false,
+              readonly: false,
+              meta: {
+                quote: undefined,
+                hasLeftSideExpression: false
+              }
+            }
+          ]
+        },
+        jsdoc: {
+          type: 'JsdocTypeObject',
+          meta: {
+            separator: 'semicolon'
+          },
+          elements: [
+            {
+              type: 'JsdocTypeKeyValue',
+              key: 'object',
+              right: {
+                type: 'JsdocTypeName',
+                value: 'string'
+              },
+              optional: false,
+              readonly: false,
+              meta: {
+                quote: undefined,
+                hasLeftSideExpression: false
+              }
+            }
+          ]
+        },
+        closure: {
+          type: 'JsdocTypeObject',
+          meta: {
+            separator: 'semicolon'
+          },
+          elements: [
+            {
+              type: 'JsdocTypeKeyValue',
+              key: 'object',
+              right: {
+                type: 'JsdocTypeName',
+                value: 'string'
+              },
+              optional: false,
+              readonly: false,
+              meta: {
+                quote: undefined,
+                hasLeftSideExpression: false
+              }
+            }
+          ]
+        }
+      },
```

Comment:
if all expected results are the same you can use `expected` instead of `diffExpected`.

```suggestion
      expected: {
        type: 'JsdocTypeObject',
        meta: {
          separator: 'semicolon'
        },
        elements: [
          {
            type: 'JsdocTypeKeyValue',
            key: 'object',
            right: {
              type: 'JsdocTypeName',
              value: 'string'
            },
            optional: false,
            readonly: false,
            meta: {
              quote: undefined,
              hasLeftSideExpression: false
            }
          }
        ]
      },
```